src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseDataFileService.cs

```diff
@@ -428,89 +432,146 @@ public async Task<Either<ActionResult, DataFileInfo>> UploadAsZip(Guid releaseVe
                 });
         }
 
-        public async Task<Either<ActionResult, List<DataFileInfo>>> UploadAsBulkZip(
+        public async Task<Either<ActionResult, List<ArchiveDataSetFile>>> ValidateAndUploadBulkZip(
```

Comment:
Need a test in `ReleaseDataFileServicePermissionTests` for this.

The existing permission test also needs to be renamed to match the new method name.